File: app/src/main/java/com/aaron/realgltextureview/SimpleEGLConfigChooser.java
package com.aaron.realgltextureview;

/**
 * This class will choose a RGB_888 surface with
 * or without a depth buffer.
 *
 */
class SimpleEGLConfigChooser extends ComponentSizeChooser {
    public SimpleEGLConfigChooser(boolean withDepthBuffer, int eglContextVersion) {
        super(8, 8, 8, 0, withDepthBuffer ? 16 : 0, 0, eglContextVersion);
    }
}

File: iglview/src/main/java/com/aaronlee/iglview/SimpleEGLConfigChooser.kt
package com.aaronlee.iglview

/**
 * This class will choose a RGB_888 surface with
 * or without a depth buffer.
 *
 */
internal class SimpleEGLConfigChooser(withDepthBuffer: Boolean, eglContextVersion: Int) :
    ComponentSizeChooser(8, 8, 8, 0, if (withDepthBuffer) 16 else 0, 0, eglContextVersion)